### ISSUE TITLE
Give different color to Git status if there are unpushed commits

### ISSUE THREAD

**Issue by rvhuang** [cmderdev/cmder#1721]
Currently with or without unpushed commits, the Git status font colors is always white (see pic). However it could be better if the color can indicate whether there are unpushed commits just like there are uncommited changes (which is red). Thanks.

![image](https://user-images.githubusercontent.com/11286168/37699190-4f04a16e-2d21-11e8-810f-071c8ba9ef3b.png)

**Comment by SuperSandro2000** [comment]
I think this is on the behalf of Git and not cmder.

**Comment by SuperSandro2000** [comment]
I don't think this is a good idea because the right pictures is quite hard to read. Maybe an option?

**Comment by lfire** [comment]
When the files were changed, the color will be red, otherwise it will be white. This user experience started from the old version(V 1.3.4), and I think should be continue.

**Comment by daxgames** [comment]
This is what I'd like to see but have not tried to figure out how to do it:

| Repo Status | Git Prompt Color |
| :------------ | :----------------- |
| Clean           |  white                  |
| Clean Not Pushed| yellow |
| Dirty(commit required) | red|

**Comment by SuperSandro2000** [comment]
I am sure that PR's are welcome

**Comment by daxgames** [comment]
They are I just don't have time to do it right now.  I put this out here as a discussion point to see if others agree.  If they do maybe I'll find the time or someone else will.

**Comment by thatsIch** [comment]
I agree, it was a very useful feature, but I have personally no clue what changed in the first place even glancing at the diffs

https://github.com/cmderdev/cmder/compare/46d468aea09d59e260c57282b6c5de0d6e9f8b29...a49c1b3f3daa8c4a35529a902840cc38648d7d9e

**Comment by sebastian-hj** [comment]
This apperas to have been fixed in [v1.3.6](https://github.com/cmderdev/cmder/releases/tag/v1.3.6).

**Comment by daxgames** [comment]
#1884 - Not exactly what I suggested but better.

* Git prompt now shows
   * Clean ->White
   * Dirty -> Yellow
   * Conflict -> Red
